client/clip_client/client.py

```diff
@@ -109,6 +110,13 @@ def encode(self, content, **kwargs):
                 f'content must be an Iterable of [str, Document], try `.encode(["{content}"])` instead'
             )
 
+        if not isinstance(content, types.GeneratorType) and len(
+            content
+        ) > 100 * kwargs.get('batch_size', 8):
+            warnings.warn(
+                f'Document size is too large, it is recommended to use smaller document size for better performance'
```

Comment:
`Document size` is misleading

Reply:
maybe we can place this logic in `self._prepare_streaming`

Reply:
ur->